delete-netlify/index.js

```diff
@@ -0,0 +1,74 @@
+const axios = require("axios");
```

Comment:
can add comments what is the pre=setup req

Reply:
Haha yea, i am trying to standardise all our tooling to have at least 1-2 lines of descriptions so that during remediations we can quickly jump to action than double checking if what we think the code does is correct

Sure just 1-2 lines comments or a README will be good!

Reply:
Thanks @seaerchin

Reply:
added